src/sentry/logging/handlers.py

```diff
@@ -0,0 +1,29 @@
+"""
+sentry.logging.handlers
+~~~~~~~~~~~~~~~~~~~~~~~
+:copyright: (c) 2010-2016 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+
+import logging
+
+
+class StructLogHandler(logging.StreamHandler):
+    def __init__(self, *args, **kwargs):
+        from structlog import get_logger
+        super(StructLogHandler, self).__init__(*args, **kwargs)
+        self._structlog = get_logger()
```

Comment:
It seems it'd be safe to make this module level then it doesn't have to be an instance on each `StructLogHandler`.

Is there a reason I'm overlooking why this is needed per-instance?